dali/operators/reader/loader/file_loader.cc

```diff
@@ -24,34 +25,53 @@
 namespace dali {
 
 
-inline void assemble_file_list(const std::string& path, const std::string& curr_entry, int label,
-                        std::vector<std::pair<std::string, int>> *file_label_pairs) {
+inline void assemble_file_list(const std::string& path, const std::string& curr_entry,
+                               const std::string& filter, std::vector<std::string>* file_list) {
```

Comment:
when you're touching this code, change pointer to reference... It's one of the few remaining places where we blindly followed rather outdated coding style and now we only use pointers where we expect either indexing or allow null.

Reply:
ok, I will update that then. I tried to keep the changes minimal to the original code, but if you want to fix that anyway I can do that.

Reply:
fixed